src/test/java/ProfileTest.java

```diff
@@ -0,0 +1,41 @@
+import Pages.HomePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import Pages.LoginPage;
+import Pages.ProfilePage;
+
+import java.net.MalformedURLException;
+import java.time.Duration;
+
+public class ProfileTest extends BaseTest {
+
+    @BeforeMethod
+    @Parameters({"BaseUrl"})
+    public void launchBrowser_main(String BaseUrl) throws MalformedURLException {
+        this.launchBrowser(BaseUrl);
+    }
+
+    @AfterMethod
+    public void closeBrowser_main() {
+        this.closeBrowser();
+    }
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

![image](https://user-images.githubusercontent.com/104761482/216757671-f66ce424-ae1d-4ffe-8673-05a0e07924ad.png)

![image](https://user-images.githubusercontent.com/104761482/216757679-85565d97-e2bb-4569-8c6f-4d5cfb5c1db5.png)